src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -28,10 +28,6 @@
 // disabled. Server GC is off by default.
 static const char* serverGcVar = "CORECLR_SERVER_GC";
 
-// Name of the environment variable controlling concurrent GC,
-// used in the same way as serverGcVar. Concurrent GC is on by default.
-static const char* concurrentGcVar = "CORECLR_CONCURRENT_GC";
```

Comment:
Windows CoreRun doesn't use the 'new' host interface (in unixinterface.cpp), and therefore doesn't pass in any configuration key/value pairs, which is the problem with the Unix one. We could probably remove the environment variable on Windows too, but it's not harming anything and I'd like to keep this change minimal since it's intended for RTM.